app/services/slack/messengers/sponsorship.rb

```diff
@@ -1,37 +0,0 @@
-module Slack
```

Comment:
I could not find a reference to this anywhere in the code, would appreciate a second pair of eyes on this please. Hence I've deleted it.